Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
-github "ReactiveX/RxSwift" "3.0.1"
-github "daltoniam/Starscream" "2.0.2"
+github "ReactiveX/RxSwift" "3.4.0"
```

Comment:
This really doesn't matter because this is `Cartfile.resolved`, not Cartfile. Version number is not set to specific one. It's different because in the meantime I was working on different branches I updated carthage frameworks on my laptop.